examples/Surface.nopadding.tsx

```diff
@@ -30,27 +30,27 @@ export default () => {
               padding: 0,
             }}
           >
-            <li style={listStyle}>
+            <li style={listStyle} tabIndex={0}>
               <Anchor>Daily log</Anchor>
             </li>
-            <Divider />
-            <li style={listStyle}>
+            <Divider as='li' />
```

Comment:
instead of turning divider into dummy list items, it would be better to just turn the `<ul>`/`<li>` elements into `<div>` and avoid the issue altogether

Reply:
Converted the list structure to`<div>` elements [here](https://github.com/iTwin/iTwinUI/pull/1500/commits/aab4ea1fa72f0a30d754c4bc2490c86ab1886bee).